### PR TITLE
feat(build): Checks config.js syntax.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

**Review by saghul — APPROVED** [review]
Legend

File: web/rootfs/defaults/settings-config.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ config.enableNoisyMicDetection = {{ $ENABLE_NOISY_MIC_DETECTION }};
 //
 
 config.p2p = {
-    enabled: {{ $ENABLE_P2P }},
+    enabled: {{ $ENABLE_P2P }};
```
Comment (jallamsetty1) [inline web/rootfs/defaults/settings-config.js:133]: Don't you have to revert this?

Comment (saghul) [reply web/rootfs/defaults/settings-config.js:133]: I suppose he is testing the CI works.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -7,6 +7,37 @@ on:
...
+      run: |
+        dpkgArch="$(dpkg --print-architecture)"
+        case "${dpkgArch##*-}" in \
+          "amd64") TPL_ARCH=amd64; S6_ARCH=amd64 ;; \
+          "arm64") TPL_ARCH=arm64; S6_ARCH=aarch64 ;; \
+          *) echo "unsupported architecture"; exit 1 ;; \
+        esac
+        wget -qO /usr/bin/tpl https://github.com/jitsi/tpl/releases/download/v1.4.0/tpl-linux-${TPL_ARCH}
+        chmod +x /usr/bin/tpl
+    - name: Run tmpl with default values for settings-config.js
```
Comment (aaronkvanmeerten) [inline .github/workflows/ci.yml:30]:
```suggestion
    - name: Run tpl with default values for settings-config.js
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -7,6 +7,37 @@ on:
...
+      run: tpl web/rootfs/defaults/settings-config.js > /tmp/settings-config.js
+    - name: Run tmpl with default values for system-config.js
```
Comment (aaronkvanmeerten) [inline .github/workflows/ci.yml:32]:
```suggestion
    - name: Run tpl with default values for system-config.js
```

File: web/rootfs/defaults/settings-config.js (hunk 1 above)
Comment (damencho) [reply web/rootfs/defaults/settings-config.js:133]: yep the commit message is drop: :)

**Comment by damencho** [comment]
image

Dropping the broken commit now :)

**Comment by damencho** [comment]
Hum ... I will need to merge both files :)

**Review by aaronkvanmeerten — APPROVED** [review]

**Pull request merged**